crates/oxc_linter/src/snapshots/prefer_each.snap

```diff
@@ -0,0 +1,218 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-vitest(prefer-each): Enforce using `each` rather than manual loops
+   ╭─[prefer_each.tsx:1:3]
+ 1 │ ╭─▶   for (const [input, expected] of data) {
+ 2 │ │                     it(`results in ${expected}`, () => {
+ 3 │ │                       expect(fn(input)).toBe(expected)
+ 4 │ │                     });
+ 5 │ ╰─▶                    }
+   ╰────
+  help: Prefer using `it.each` rather than a manual loop.
+
+  ⚠ eslint-plugin-vitest(prefer-each): Enforce using `each` rather than manual loops
+   ╭─[prefer_each.tsx:1:2]
+ 1 │ ╭─▶  for (const [input, expected] of data) {
+ 2 │ │                     describe(`when the input is ${input}`, () => {
+ 3 │ │                       it(`results in ${expected}`, () => {
+ 4 │ │                      expect(fn(input)).toBe(expected)
+ 5 │ │                       });
+ 6 │ │                     });
+ 7 │ ╰─▶                    }
+   ╰────
+  help: Prefer using `describe.each` rather than a manual loop.
+
+  ⚠ eslint-plugin-vitest(prefer-each): Enforce using `each` rather than manual loops
+   ╭─[prefer_each.tsx:1:1]
+ 1 │ ╭─▶ for (const [input, expected] of data) {
```

Comment:
Consider only covering the test part of the for-loop with your spans. These are quite large.